pykafka/producer.py

```diff
@@ -241,3 +237,326 @@ def produce(self, messages):
         # only *some* messages when a leader changes. Therefore, we don't want
         # a random partition distribution changing that on the retry.
         self._produce(self._partition_messages(messages), 0)
+
+
+_QueueInfo = namedtuple('QueueInfo', ['lock', 'flush_ready',
+                                      'slot_available', 'queue',
+                                      'producer'])
+
+
+class QueueInfo(_QueueInfo):
```

Comment:
It's not really "info" is it? It's really an async queue, since it contains the deque we're using as a queue.